sktm/patchwork.py

```diff
@@ -39,6 +38,11 @@
     r"pull.?request"
 ]
 
+PW_CHECK_CHIOCES = {'pending': 0,
```

Comment:
Fixing the typo. Vim's autofill isn't helping and introduced it everywhere :D

Yes, `check` is how it's officially called in Patchwork. The options in the (now) dictionary are marked as `choices` in the API description, because, well, you have to choose from those values (hence why I decided to call them that).

I don't see any benefit of enums in Python - enum basically is the same thing as namedtuple or dict, mapping a name to a value and nothing else. And as you can see in the changed parts of code, you have to explicitly type the enumed thing to `int` to get the value, while with the dict or namedtuple you simply access it. Why install and import something, and have to remember to type it, to get the same functionality you can achieve by using standard library? TBH I'd like to get rid of enums in skt/sktm eventually.